src/main/kotlin/minesweeper/domain/BoardSize.kt

```diff
@@ -0,0 +1,21 @@
+package minesweeper.domain
+
+class BoardSize(private val width: Int, private val height: Int) {
+
+    init {
+        require(value = width in BOARD_SIZE_RANGE && height in BOARD_SIZE_RANGE) {
+            "점은 $BOARD_SIZE_RANGE 사이에 위치해야 합니다. 입력값 : $width, $height"
+        }
+    }
+
+    fun area(): Int = width * height
```

Comment:
[코틀린 코드 컨벤션](https://kotlinlang.org/docs/coding-conventions.html#choose-good-names)에서는 메서드의 네이밍을 `동사` or `동사구`를 사용하라고 가이드하고있어요! 👀

지금과 같이 프로퍼티를 호출하듯이 사용하고싶다면 아래와 같은 방식으로 변경해볼 수도 있을 것 같아요. 😃

```suggestion
    val area: Int
        get() = width * height
```

Reply:
수정했습니다. 😊